stackslib/src/net/stackerdb/sync.rs

```diff
@@ -249,11 +249,9 @@ impl<NC: NeighborComms> StackerDBSync<NC> {
             .get_slot_write_timestamps(&self.smart_contract_id)?;
 
         if local_slot_versions.len() != local_write_timestamps.len() {
-            // interleaved DB write?
-            return Err(net_error::Transient(
-                "Interleaved DB write has led to an inconsistent view of the stackerdb. Try again."
-                    .into(),
-            ));
+            let msg = format!("Local slot versions ({}) out of sync with DB slot versions ({}); abandoning sync and trying again", local_slot_versions.len(), local_write_timestamps.len());
+            warn!("{}", &msg);
+            return Err(net_error::Transient(msg));
```

Comment:
@jcnelson can you confirm that the master versions of the `stackerdb::sync` changes should be preferred here?

Reply:
Yes